services/Network/expressRouteCircuits/alerts.yaml

```diff
@@ -189,60 +189,3 @@
     criterionType: StaticThresholdCriterion
     threshold: 800000000.0
     autoMitigate: false
```

Comment:
@Alboroni and @JoeyBarnes I originally deleted these because we already had defined them in the ALZ pattern.